back/engines/commercial/id_franceconnect/config/initializers/httpclient_patch.rb

```diff
@@ -1,20 +0,0 @@
-# frozen_string_literal: true
-
-# Copied from https://gitlab.com/gitlab-org/gitlab-foss/-/merge_requests/30749
-
-# By default, httpclient (and hence anything that uses rack-oauth2)
-# ignores the system-wide SSL certificate configuration in favor of its
-# own cacert.pem. This makes it impossible to use custom certificates
-# without patching that file. Until
-# https://github.com/nahi/httpclient/pull/386 is merged, we work around
-# this limitation by forcing the HTTPClient SSL store to use the default
-# system configuration.
-module HTTPClient::SSLConfigDefaultPaths
```

Comment:
Why do you remove it? Did you manage to reproduce the original issue in the old version and fix it in the new version? https://github.com/CitizenLabDotCo/citizenlab/pull/1590

Reply:
But anyway it may be useful to test that the original issue is still reproduced in master, but not in this branch.